meta/sqlite/gauge.go

```diff
@@ -61,11 +62,11 @@ END;
 var constituent = Table{
 	Create: constituentCreate,
 	Insert: func() string {
-		return fmt.Sprintf("INSERT INTO constituent (gauge, number, constituent, amplitude, lag, start_date, end_date) VALUES ((%s), ?, ?, ?, ?, ?, ?);",
+		return fmt.Sprintf("INSERT INTO constituent (gauge_id, location, number, constituent, amplitude, lag, start_date, end_date) VALUES ((%s), ?, ?, ?, ?, ?, ?, ?);",
 			gauge.Select(),
 		)
 	},
-	Fields: []string{"Gauge", "Identification Number", "Analysis Time Zone", "Analysis Latitude", "Analysis Longitude", "Crex Tag", "Start Date", "End Date"},
+	Fields: []string{"Gauge", "Location", "Number", "Constituent", "Amplitude", "Lag", "Start Date", "End Date"},
```

Comment:
Though the new field names looks more like what we've been using, not sure where the old one applied and why changed now?

Reply:
This basically gives a list of csv "column" names to load into the sqlite database when inserting from the raw files.